src/gui/accountsettings.cpp

```diff
@@ -19,6 +19,8 @@
 
 #include "accountsettings.h"
 
+#include "../agent.h"
```

Comment:
are relative filepaths like this okay?

Reply:
yes, it shows the close relationship between that header and the GUI.